File: update_color.py
import os

file_path = r'C:\Users\cerc\Documents\GitHub\flynow\assets\css\app.css'

try:
    with open(file_path, 'r', encoding='utf-8') as f:
        content = f.read()
    
    new_content = content.replace('#f33c0e', '#FF6600')
    
    with open(file_path, 'w', encoding='utf-8') as f:
        f.write(new_content)
        
    print("Successfully updated color.")
except Exception as e:
    print(f"Error: {e}")

File: update_color.js
const fs = require('fs');
const path = 'assets/css/app.css';
try {
  if (fs.existsSync(path)) {
    let content = fs.readFileSync(path, 'utf8');
    content = content.replace(/#f33c0e/g, '#FF6600');
    fs.writeFileSync(path, content, 'utf8');
    console.log('Success');
  } else {
    console.log('File not found');
  }
} catch (e) {
  console.error(e);
}